File: simplesearchview/src/main/java/com/ferfalk/simplesearchview/utils/SimpleAnimationListener.java
package com.ferfalk.simplesearchview.utils;

import androidx.annotation.NonNull;
import android.view.View;

/**
 * @author Fernando A. H. Falkiewicz
 */
public abstract class SimpleAnimationListener implements SimpleAnimationUtils.AnimationListener {
    @Override
    public boolean onAnimationStart(@NonNull View view) {
        // No action
        return false;
    }

    @Override
    public boolean onAnimationEnd(@NonNull View view) {
        // No action
        return false;
    }

    @Override
    public boolean onAnimationCancel(@NonNull View view) {
        // No action
        return false;
    }
}

File: simplesearchview/src/main/java/com/ferfalk/simplesearchview/utils/SimpleAnimationListener.kt
package com.ferfalk.simplesearchview.utils

import android.view.View

/**
 * @author Fernando A. H. Falkiewicz
 */
abstract class SimpleAnimationListener : SimpleAnimationUtils.AnimationListener {
    override fun onAnimationStart(view: View): Boolean {
        // No action
        return false
    }

    override fun onAnimationEnd(view: View): Boolean {
        // No action
        return false
    }

    override fun onAnimationCancel(view: View): Boolean {
        // No action
        return false
    }
}